packages/Link/index.js

```diff
@@ -27,7 +27,7 @@ export const Link = forwardRef(({ children, variant = 'primary', ...props }, ref
       }
       return cloneElement(child, {
         color: theme.links.default.color,
-        fontWeight: theme.links.default.fontWeight,
+        fontWeight: child.props.variant ? null : theme.links.default.fontWeight,
```

Comment:
```suggestion
        fontWeight: child.props.variant ? undefined : theme.links.default.fontWeight,
```
undefined est plus adapté puisque ce n'est pas un objet non  ?